ext/nnstreamer/tensor_filter/tensor_filter_nnfw.c

```diff
@@ -396,10 +394,21 @@ nnfw_tensor_info_set (const nnfw_pdata * pdata,
   for (idx = nnfw_info.rank - 1; idx >= 0; idx--)
     nnfw_info.dims[nnfw_info.rank - idx - 1] = info->dimension[idx];
 
-  for (idx = NNFW_TENSOR_RANK_LIMIT - 1; idx >= nnfw_info.rank; idx--)
+  /** @note Maximum rank expressible with nnfw is 6 (NNFW_MAX_RANK) */
+  for (idx = NNFW_MAX_RANK - 1; idx >= nnfw_info.rank; idx--)
     nnfw_info.dims[idx] = 0;
 
-  nnfw_apply_tensorinfo (pdata->session, tensor_idx, nnfw_info);
+#if defined (__TIZEN__)
```

Comment:
What about https://mesonbuild.com/Compiler-properties.html#does-a-function-exist ?

Reply:
add todo and new definition (NNFW_USE_OLD_API) for this case